static/app/views/replays/detail/tagPanel/index.spec.tsx

```diff
@@ -14,7 +14,7 @@ const mockReplay = ReplayReader.factory({
     },
     tags: {
       foo: ['bar', 'baz'],
-      'my custom tag': ['a wordy value'],
+      my_custom_tag: ['a wordy value'],
```

Comment:
why did this have to change?

Reply:
without the tags[] wrapper, "my custom tag" is interpreted as 3 separate tokens. So in the searchbar the query is "tag": "a wordy value", and "my" "custom" just floating there

Reply:
Maybe we can wrap it with quotes. Something to look into